src/components/home.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import'./index.css';
+
+
+export const Home = (props) =>{
+  return (
+    <div className = "text">
+    <h1>Pick A List</h1>
+    <div>
+    <MainArticle 
+    title = "Categories"
+    />
+    </div>
+    <div className="home">
+    <Article  
+    Img = "https://res.cloudinary.com/cn-project/image/upload/v1641481525/pana/Winter_solstice-pana_x4wbwx.png" 
+    alt= "Animated picture of the world globe"
+    title = "Travel"
+    // category= "Travel" 
```

Comment:
do we need to keep these comments?

Reply:
The comments were placeholders for us to discuss what else needs to go into this section together but can be removed once they have been decided on